dask/core.py

```diff
@@ -37,7 +37,7 @@ def istask(x):
     >>> istask(1)
     False
     """
-    return type(x) is tuple and x and callable(x[0])
+    return type(x) is tuple and x and (callable(x[0]) or hasattr(x[0], "_callable"))
```

Comment:
What was this for? Ideally I'd like to not have to merge this...

Reply:
Oops (from debugging serialization)